client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -62,7 +66,9 @@ class RegistrantExtraInfoFrForm extends React.PureComponent {
 
 	render() {
 		const translate = this.props.translate;
-		const registrantType = get( this.props.contactDetails, 'extra.registrantType', defaultRegistrantType );
+		const { contactDetails } = this.props;
+		const registrantType = get( contactDetails, 'extra.registrantType', defaultRegistrantType );
```

Comment:
Eh. 😆  (It's fine. Is that idiomatic? Like never reference beyond `this.props` directly?)

Reply:
No, I just had code here using the `contactDetails` at one point, and forgot to fix it afterwards :/

I have gotten feedback to favour `{ contactDetails } = this.props` over `this.props.contactDetails`, though, and I've grown to appreciate having the props unpacked like that first (it makes it more obvious how to use the component), so I think I will fix this rather than revert it.